addons/stock/models/stock_move.py

```diff
@@ -387,8 +387,24 @@ def name_get(self):
                 move.location_id.name, move.location_dest_id.name)))
         return res
 
+    def set_move_name_in_vals(self, vals):
+        """Set `name` field from supplied the supplied vals, if `product_id` is set"""
```

Comment:
The docstring doesnt seem very explanatory to me, maybe is because of my English :)

Reply:
Will improve this if we keep the change. :)

Reply:
Change is gonna be dropped and will take the docstring with it. :P

Reply:
Dropped. :)